rules/use-apps-for-expenses/rule.md

```diff
@@ -143,7 +72,7 @@ The best feature of Xero Expense reimbursement is that the employee's bank detai
 
 
 
-Although Expensify and Receipt Bank are usable too for this purpose, their costs cannot compare to Xero Expense app for this purpose.
+Although Expensify and Receipt Bank are capable of performing this same function, the cost to have the same function is expensive.
```

Comment:
What do you think of this wording:

Although Expensify and Receipt Bank are capable of performing this function, it is expensive.

Reply:
Done